catalogue/webapp/services/wellcome/content/types/index.ts

```diff
@@ -0,0 +1,41 @@
+import { DigitalLocation, Contributor } from '@weco/common/model/catalogue';
+import { Content, ContentApiProps } from './api';
+
+export type { ContentApiProps };
+
+// Response objects
+export type ContentApiError = {
+  errorType: string;
+  httpStatus: number;
+  label: string;
+  description: string;
+  type: 'Error';
+};
+
+export type Image = {
+  type: 'Image';
+  id: string;
+  locations: DigitalLocation[];
+  source: {
+    id: string;
+    title: string;
+    contributors?: Contributor[];
+    type: string;
+  };
+  visuallySimilar?: Image[];
+  aspectRatio?: number;
+};
+
+export type ContentResultsList<Result extends Content> = {
+  type: 'ResultList';
+  totalResults: number;
+  totalPages: number;
+  results: Result[];
+  pageSize: number;
+  prevPage: string | null;
+  nextPage: string | null;
+
+  // We include the URL used to fetch data from the content API for
+  // debugging purposes.
+  _requestUrl: string;
+};
```

Comment:
Possibly not right now, but this is the sort of structure which is shared between catalogue/content API and could be a generic "Wellcome" type.